src/components/SearchBar.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import styles from '../styles/SearchBar.module.css';
+
+export default function SearchBar({ onSearch }) {
+    const [searchInput, setSearchInput] = useState('');
+    const [loginErrorMessage, setLoginErrorMessage] = useState('');
+
+    const handleInputChange = (e) => {
+        setSearchInput(e.target.value);
+    };
+    
+    const handleSearch = (e) => {
+        let accessToken = localStorage.getItem('access_token');
+
+        if (accessToken && searchInput) {
```

Comment:
The accessToken check here seems redundant as you're already checking it in  httpRequests.search(), can you remove it? It's possible you need the check for a better user experience, I just wanted to double check that it was necessary

Reply:
Ah, good point. I've removed the accessToken check from the handleSearch and handleKeyDown functions.